### PR TITLE
Remove unnecessary build() on attachDbPRequest

### PR DESCRIPTION
**Pull request by ZacBlanco**

### CODE REVIEW

File: experimental/table/client/src/main/java/alluxio/client/catalog/RetryHandlingCatalogMasterClient.java [hunk 1]
```diff
@@ -105,7 +105,7 @@ public TableInfo getTable(String databaseName, String tableName) throws AlluxioS
   public boolean attachDatabase(String dbName, Map<String, String> configuration)
       throws AlluxioStatusException {
     return retryRPC(() -> mClient.attachDatabase(
-        AttachDatabasePRequest.newBuilder().build().newBuilder().setDbName(dbName)
+        AttachDatabasePRequest.newBuilder().setDbName(dbName)
```
Comment (gpang) [inline experimental/table/client/src/main/java/alluxio/client/catalog/RetryHandlingCatalogMasterClient.java]: you will have to rebase, to get the updated line.

Comment (ZacBlanco) [reply experimental/table/client/src/main/java/alluxio/client/catalog/RetryHandlingCatalogMasterClient.java]: This was on the latest master though? Did you fix it in another branch?

Comment (ZacBlanco) [reply experimental/table/client/src/main/java/alluxio/client/catalog/RetryHandlingCatalogMasterClient.java]: whoops nevermind. I guess you changed it around when I submitted the PR

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**